lib/kvs/kvs_cursor.c

```diff
@@ -452,29 +454,33 @@ ikvs_curcache_insert(struct curcache_bucket *bkt, struct kvs_cursor_impl *cur)
                 cur->kci_item.ci_next = old;
                 bkt->cb_active++;
                 *pp = cur;
+                cur = NULL;
                 break;
             }
 
             pp = &old->kci_item.ci_next;
         }
 
-        if (++entry->ce_cnt > 8 || bkt->cb_active > bkt->cb_entrymax) {
+        if (++entry->ce_cnt > 64 || bkt->cb_active > bkt->cb_entrymax) {
```

Comment:
I would prefer if this `64` was a `#define` somewhere.

Reply:
Will add one in my next PR.